src/internal.c

```diff
@@ -10240,6 +10242,9 @@ static int DoCertificateStatus(WOLFSSL* ssl, byte* input, word32* inOutIdx,
     if (ret != 0)
         SendAlert(ssl, alert_fatal, bad_certificate_status_response);
 
+    if (IsEncryptionOn(ssl, 0))
+        *inOutIdx += ssl->keys.padSz;
```

Comment:
Is a size check needed here?

Reply:
It is a good idea. For this scenario, it isn't checked in every case. I'll go back and add the checks. (DoHelloRequest is doing the size check for the pad.)